README.md

```diff
@@ -39,31 +39,31 @@ $ brew remove eosio
 ```
 #### Ubuntu 18.04 Debian Package Install
 ```sh
-$ wget https://github.com/eosio/eos/releases/download/v1.6.0/eosio_1.7.0-rc2-ubuntu-18.04_amd64.deb
-$ sudo apt install ./eosio_1.7.0-rc2-ubuntu-18.04_amd64.deb
+$ wget https://github.com/eosio/eos/releases/download/v1.7.0/eosio_1.7.0-ubuntu-18.04_amd64.deb
```

Comment:
I think these links need to be `EOSIO` instead of `eosio`

```
$ curl -I https://github.com/eosio/eos/releases/download/v1.6.0/eosio_1.6.0-ubuntu-18.04_amd64.deb 
HTTP/1.1 404 Not Found
```
vs
```
$ curl -I https://github.com/EOSIO/eos/releases/download/v1.6.0/eosio_1.6.0-1-ubuntu-18.04_amd64.deb
HTTP/1.1 302 Found
```

Reply:
ah no, it's the removal of -1- that is different

Reply:
This worked for me: curl -I https://github.com/eosio/eos/releases/download/v1.6.3/eosio-1.6.3-1.el7.x86_64.rpm